docs/source/whats_new/v1.2.0.txt

```diff
@@ -0,0 +1,84 @@
+.. currentmodule:: databroker
+
+v1.2.0 (2020-12-17)
+-------------------
+
+Added
++++++
+
+* New accessors on ``BlueskyEventStream`` named ``config``, ``timestamps``, and
+  ``config_timestamps`` provide xarray Datasets of secondary readings from
+  Event Descriptors and Events.
+
+  .. code:: python
+
+     >>> run.primary
+     <BlueskyEventStream 'primary' from Run 69fd42fa...>
+
+     >>> run.primary.read()
+     <xarray.Dataset>
+     Dimensions:  (time: 3)
+     Coordinates:
+     * time     (time) float64 1.608e+09 1.608e+09 1.608e+09
+     Data variables:
+         det      (time) float64 1.0 1.0 1.0
+
+     >>> run.primary.timestamps.read()
+     <xarray.Dataset>
+     Dimensions:  (time: 3)
+     Coordinates:
+       * time     (time) float64 1.608e+09 1.608e+09 1.608e+09
+     Data variables:
+         det      (time) float64 1.608e+09 1.608e+09 1.608e+09
+
+     >>> list(run.primary.config)
+     ['det']
+
+     >>> run.primary.config.det.read()
+     <xarray.Dataset>
+     Dimensions:               (time: 3)
+     Coordinates:
+       * time                  (time) float64 1.608e+09 1.608e+09 1.608e+09
+     Data variables:
+         det_Imax              (time) int64 1 1 1
+         det_center            (time) int64 0 0 0
+         det_sigma             (time) int64 1 1 1
+         det_noise             (time) <U4 'none' 'none' 'none'
+         det_noise_multiplier  (time) int64 1 1 1
+
+     >>> run.primary.config_timestamps.det.read()
+     <xarray.Dataset>
+     Dimensions:               (time: 3)
+     Coordinates:
+       * time                  (time) float64 1.608e+09 1.608e+09 1.608e+09
+     Data variables:
+         det_Imax              (time) float64 1.608e+09 1.608e+09 1.608e+09
+         det_center            (time) float64 1.608e+09 1.608e+09 1.608e+09
+         det_sigma             (time) float64 1.608e+09 1.608e+09 1.608e+09
+         det_noise             (time) float64 1.608e+09 1.608e+09 1.608e+09
+         det_noise_multiplier  (time) float64 1.608e+09 1.608e+09 1.608e+09
```

Comment:
It's an impressive feature, but should it rather go to the general documentation rather than the release notes?

Reply:
I see your point. I've been in the habit of showing quick examples in the release notes so that it's easy to quickly get up to speed on new features. :shrug:

Reply:
OK, sounds good.